### PR TITLE
Task 4103 - Vuetify Bottomsheets refactoring

### PR DESCRIPTION
**Pull request by JungPham**
- [x] remove extends of Sheet class, as elevation and rounded methods are not applicable
- [x] create custom classes for BottomSheets from the page
- [x] change tests to use custom classes instead of dynamic find like `listBottomSheet.openInOptionsList().get(1).find("img").click();`
- [x]  Fix comments from first review

### CODE REVIEW

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java [hunk 1]
```diff
@@ -102,19 +102,20 @@ public void playerBottomSheetTest() {
...
     @Test
-    public void listBottomSheetTest(){
+    public void listBottomSheetTest() {
         listBottomSheet.is().hidden();
 
         listBottomSheetButton.click();
         listBottomSheet.is().displayed();
-        listBottomSheet.openInOptionsList().get(1).find(".v-list-item__title").has().text("Keep");
-        listBottomSheet.openInOptionsList().get(1).find("img").click();
+        listBottomSheet.getOptionTitle(1, "Keep");
```
Comment (behhVenn) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java:110]: what the reason to keep text by index? You can implement isOptionTitlesPresented() and verify into it that all test titles are presented and verify return value in the test.

Comment (behhVenn) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java:110]: list of test values can be defined in test level or in page object

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java [hunk 2]
```diff
@@ -102,19 +102,20 @@ public void playerBottomSheetTest() {
...
+        listBottomSheet.clickOption(1);
         listBottomSheet.is().hidden();
 
+
```
Comment (behhVenn) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java:114]: what's the reason to recheck element one more time? You can check all available options or first and last points as minimum

**Comment by behhVenn** [comment]
fix tests run issue please

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java (hunk 2 above)
Comment (JungPham) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/composite/BottomSheetsTests.java:114]: Thanks for your help, I will check first and last points

**Review by vklonin — APPROVED** [review]

**Pull request merged**